css-gaps-1/Overview.bs

```diff
@@ -0,0 +1,785 @@
+			or adjacent to a child item spanning across the gap per the steps below.
+		</dd>
+		<dt><dfn>none</dfn>
+		<dd>
+			Gap decorations start and end at <a>gap intersection points</a>
+			which are at an edge of the container only.
+		</dd>
+	</dl>
+
+	Issue: Bikeshed these values.
+
+	<pre class='propdef shorthand'>
+		Name: gap-rule-break
+		Value: <<'column-rule-break'>>
+		Applies to: Same as 'column-rule-break' and 'row-rule-break'
+	</pre>
+
+	Sets the 'column-rule-break' and 'row-rule-break' properties to the same value.
+
+	To <dfn>determine pairs of gap decoration endpoints</dfn> within a given |gap|:
+
+	<ol>
+		<li>
+			Let |pairs| be an empty list.
+			Let |endpoints| be the list of <a>gap intersection points</a> that fall along the centerline of |gap|,
+			ordered from start to end along |gap|'s axis.
+			Let |break| be the used value of either 'column-rule-break' or 'row-rule-break',
+			whichever applies to |gap|.
+		</li>
+
+		<li>
+			If |endpoints| contains fewer than 2 items, return |pairs|.
+		</li>
+
+		<li>
+			Remove the first item from |endpoints|.
+			Let |start| be equal to that item.
+		</li>
+
+		<li>
+			Remove the first item from |endpoints|.
+			Let |end| be equal to that item.
+		</li>
+
+		<li>
+			If |endpoints| is non-empty, and either of the following conditions is true...
+			<ol>
+				<li>
+					|break| is ''column-rule-break/spanning-item'',
+					and a line segment from |start| to the first item in |endpoints|,
+					with the same width as the |gap|,
+					does not intersect a child item in the container
+				</li>
+				<li>
+					|break| is ''column-rule-break/none''
+				</li>
+			</ol>
+			...then go back to step 4.
+
+		<li>
+			<a>Compute the offset</a> for |start| within |gap|.
+			Offset |start| by the result,
+			in the forward direction along |gap|'s axis.
+		</li>
+
+		<li>
+			<a>Compute the offset</a> for |end| within the |gap|.
+			Offset |end| by the result,
+			in the reverse direction along |gap|'s axis.
+		</li>
+
+		<li>
+			Add to |pairs| a tuple consisting of |start| and |end|.
+		</li>
+
+		<li>
+			Go back to step 2.
+		</li>
+	</ol>
+
+	A <a>gap decoration</a> is painted between each pair of endpoints identified using the steps above.
+
+	The following examples illustrate various settings for the <a href="#break">*-rule-break</a> properties.
+	To make the differences more apparent, the <a href="#outset">*-rule-outset</a> properties are set to ''0''.
+
+	<div class="example">
+		<pre>
+			.break-intersection {
+				gap-rule-break: intersection;
+				grap-rule-outset: 0px;
+			}
+		</pre>
+		<img src="images/example-break-intersection.png">
+		<figcaption>
+			<a>Gap decorations</a> broken at every <a>gap intersection point</a>.
+		</figcaption>
+	</div>
+
+	<div class="example">
+		<pre>
+			.break-spanning-item {
+				gap-rule-break: spanning-item;
+				grap-rule-outset: 0px;
+			}
+		</pre>
+		<img src="images/example-break-spanning-item.png">
```

Comment:
Just realized there's an error in this image: The gap decorations should not be drawn through the intersection neighbored by 4, 6, 7, and 8. Will fix.

Reply:
Hmm, actually never mind, as defined by the algorithm, the decorations *should* be drawn through that intersection.